backend/audit/templates/audit/my_submissions.html

```diff
@@ -59,71 +61,122 @@ <h2>Create a new audit</h2>
                 <input type="submit" id="start-submission" class="usa-button" value="Start a new submission" disabled />
             </div>
         </form>
+        </form>
```

Comment:
What are all of these HTML updates? At the very least, this is a redundant addition of the closing `form` tag.